app/helpers/catalog_helper.rb

```diff
@@ -12,4 +12,8 @@ def render_document_heading(*args)
     val = Array.wrap(presenter(document).heading).map { |v| h(v) }.join("<br />")
     content_tag(tag, val, { itemprop: "name", dir: val.to_s.dir }, false)
   end
+
+  def active_sort_fields_figgy
```

Comment:
Can you add a comment about why this is necessary or at least what it's doing? I get the idea reading it, but it takes some mental parsing.

Reply:
These are the options in the select_tag for the sort field: 
["score desc, updated_at_dtsi desc", "title_ssort asc", "title_ssort desc", "created_at_dtsi desc", "created_at_dtsi asc", "updated_at_dtsi desc", "updated_at_dtsi asc"]

in catalog/_members_collection.html.erb the method active_sort_fields will return:
#<Blacklight::Configuration::SortField label=\"title (A-Z)\", key=\"title_ssort asc\", field=\"title_ssort asc\", if=true, unless=false, sort=\"title_ssort asc\">","rows":"10","f.display_language_ssim.facet.limit":"6","f.state_ssim.facet.limit":"6","qf":["identifier_tesim","title_ssim","title_tesim","transliterated_title_ssim","transliterated_title_tesim","source_metadata_identifier_ssim","local_identifier_ssim","barcode_ssim"],"facet":"true","wt":"json","f.human_readable_type_ssim.facet.limit":"6"}},"error":{"metadata":["error-class","org.apache.solr.common.SolrException","root-error-class","org.apache.solr.common.SolrException"],"msg":"Can't determine a Sort Order (asc or desc) in sort spec '#<Blacklight::Configuration::SortField label=\"title (A-Z)\", key=\"title_ssort asc\", field=\"title_ssort asc\", if=true, unless=false, sort=\"title_ssort asc\">
This is why it errors.

I replaced the method with a new one 'active_sort_fields_figgy' which will return the keys we want. I build the array so that the result would be what we want in the sort field.

After the checkin meeting I looked into the catalog_controller and there is a configuration for the sort_field. We use it in the index view.

Reply:
https://github.com/projectblacklight/blacklight/blob/240013f81fa04a2663373d38a66273edfe4712bc/app/helpers/blacklight/configuration_helper_behavior.rb#L12

https://github.com/projectblacklight/blacklight/blob/7d9ff392dd6fec3c9e87b4b5b0950c77fc92aa95/spec/helpers/blacklight/configuration_helper_behavior_spec.rb#L20

Reply:
So there's a problem with title_ssort?